### PR TITLE
fix(route): update v2/scitation to v2/aip

### PR DESCRIPTION
**Pull request by Derekmini**
## 该 PR 相关 Issue / Involved Issue

Close #

## 路由地址示例 / Example for the Proposed Route(s)

```routes
/aip/aapt/ajp
```

## 新 RSS 路由检查表 / New RSS Route Checklist
  
- [ ] 新的路由 New Route
  - [ ] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [ ] 文档说明 Documentation
  - [ ] 中文文档 CN
  - [ ] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

**Review by TonyRL — COMMENTED** [review]
Please update `radar.js` as well.

**Comment by Derekmini** [comment]
> Please update `radar.js` as well.

I have already updated `radar.js`.

File: lib/v2/scitation/radar.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ module.exports = {
             {
                 title: 'journal',
                 docs: 'https://docs.rsshub.app/journal.html#scitation',
-                source: ':pub.scitation.org/toc/:jrn',
+                source: 'pubs.aip.org/:pub/:jrn',
```
Comment (TonyRL) [inline lib/v2/scitation/radar.js:8]: The `source` field could not match any domains.
docs: https://docs.rsshub.app/joinus/new-radar.html

Comment (Derekmini) [reply lib/v2/scitation/radar.js]: I have updated `radar.js`.

File: lib/v2/scitation/radar.js [hunk 2]
```diff
@@ -1,11 +1,11 @@
 module.exports = {
-    'scitation.org': {
+    'pubs.aip.org': {
```
Comment (TonyRL) [inline lib/v2/scitation/radar.js:2]: Top level object key can not match subdomain.

Comment (Derekmini) [reply lib/v2/scitation/radar.js]: Oh, I see. The modification should be as follows.

**Comment by Derekmini** [comment]
Hi @TonyRL !
By the way, the web domain of this route has been changed from `scitation.org` to `aip.org`, and the website now uses the new name `AIP` to promote the new organization. Should I move the folder `/lib/v2/scitation` to `lib/v2/aip`? Or should I keep both folders and update the docs to reflect the new one?

**Comment by TonyRL** [comment]
> Hi TonyRL ! By the way, the web domain of this route has been changed from `scitation.org` to `aip.org`, and the website now uses the new name `AIP` to promote the new organization. Should I move the folder `/lib/v2/scitation` to `lib/v2/aip`? Or should I keep both folders and update the docs to reflect the new one?

Then it would be better to update the folder and the docs as well.

**Comment by Derekmini** [comment]
> > Hi TonyRL ! By the way, the web domain of this route has been changed from `scitation.org` to `aip.org`, and the website now uses the new name `AIP` to promote the new organization. Should I move the folder `/lib/v2/scitation` to `lib/v2/aip`? Or should I keep both folders and update the docs to reflect the new one?
> 
> Then it would be better to update the folder and the docs as well.

Okay, I will do it in the next pr. Meanwhile, the `section.js` will be removed for the new website.

**Comment by TonyRL** [comment]
> > > Hi TonyRL ! By the way, the web domain of this route has been changed from `scitation.org` to `aip.org`, and the website now uses the new name `AIP` to promote the new organization. Should I move the folder `/lib/v2/scitation` to `lib/v2/aip`? Or should I keep both folders and update the docs to reflect the new one?
> > 
> > 
> > Then it would be better to update the folder and the docs as well.
> 
> Okay, I will do it in the next pr. Meanwhile, the `section.js` will be removed for the new website.

I would prefer having it done in one PR.

**Comment by Derekmini** [comment]
> > > > Hi TonyRL ! By the way, the web domain of this route has been changed from `scitation.org` to `aip.org`, and the website now uses the new name `AIP` to promote the new organization. Should I move the folder `/lib/v2/scitation` to `lib/v2/aip`? Or should I keep both folders and update the docs to reflect the new one?
> > > 
> > > 
> > > Then it would be better to update the folder and the docs as well.
> > 
> > 
> > Okay, I will do it in the next pr. Meanwhile, the `section.js` will be removed for the new website.
> 
> I would prefer having it done in one PR.

OK, I will submit pr as required as soon as possible.

**Comment by Derekmini** [comment]
Hi @TonyRL, I have updated the folder and the docs files.

File: website/docs/routes/journal.md [hunk 3]
```diff
@@ -394,20 +410,8 @@ You can get all short name of a journal from <https://www.nature.com/siteindex>
 
 ## ScienceDirect {#sciencedirect}
 
-### Journal {#sciencedirect-journal}
-
```
Comment (TonyRL) [inline website/docs/routes/journal.md:398]: This subtitle belongs to other feed.

Comment (Derekmini) [reply website/docs/routes/journal.md]: Sorry about it, I didn't do a good job of reviewing the content in the changed files.

Comment (Derekmini) [reply website/docs/routes/journal.md]: > This subtitle belongs to other feed.
Sorry about it, I didn't do a good job of reviewing the content in the changed files.

Comment (Derekmini) [reply website/docs/routes/journal.md]: > This subtitle belongs to other feed.

Sorry about it, I didn't do a good job of reviewing the content in the changed files.

File: lib/v2/aip/radar.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'aip.org': {
+        _name: 'AIP',
```
Comment (TonyRL) [inline lib/v2/aip/radar.js:3]:
```suggestion
        _name: 'American Institute of Physics',
```

**Pull request merged**